### ISSUE TITLE
better explanations and names for processes output

### ISSUE THREAD

**Issue by andrei-lihu** [codenotary/immudb#110]
We need to change some env variables names and the output should be more descriptive.

IMMU_AUTH, IMMU_ADDRESS into:

IMMUDB_SERVER_AUTH, etc

**Comment by padurean** [comment]
@andrei-lihu 
Right now these are env vars as described in the commands help:

```
➤ ./immuclient --help
CLI client for the ImmuDB tamperproof database.

Environment variables:
  IMMUCLIENT_ADDRESS=127.0.0.1
  IMMUCLIENT_PORT=3322
  IMMUCLIENT_AUTH=false
```

```
➤ ./immugw --help
Immu gateway is a smart REST proxy for the ImmuDB tamperproof database.
It exposes all gRPC methods with a REST interface while wrapping all SAFE endpoints with a verification service.

Environment variables:
  IMMUGW_ADDRESS=127.0.0.1
  IMMUGW_PORT=3323
  IMMUGW_IMMUDBADDRESS=127.0.0.1
  IMMUGW_IMMUDBPORT=3322
  IMMUGW_PIDFILE=
  IMMUGW_LOGFILE=
  IMMUGW_MTLS=false
  IMMUGW_SERVERNAME=localhost
  IMMUGW_CERTIFICATE=./tools/mtls/4_client/certs/localhost.cert.pem
  IMMUGW_PKEY=./tools/mtls/4_client/private/localhost.key.pem
  IMMUGW_CLIENTCAS=./tools/mtls/2_intermediate/certs/ca-chain.cert.pem
```


```./immudb --help
The ImmuDB tamperproof database server.

Environment variables:
  IMMUDB_DIR=.
  IMMUDB_NETWORK=tcp
  IMMUDB_ADDRESS=127.0.0.1
  IMMUDB_PORT=3322
  IMMUDB_DBNAME=immudb
  IMMUDB_PIDFILE=
  IMMUDB_LOGFILE=
  IMMUDB_MTLS=false
  IMMUDB_AUTH=false
  IMMUDB_PKEY=./tools/mtls/3_application/private/localhost.key.pem
  IMMUDB_CERTIFICATE=./tools/mtls/3_application/certs/localhost.cert.pem
  IMMUDB_CLIENTCAS=./tools/mtls/2_intermediate/certs/ca-chain.cert.pem
```

**Comment by mmeloni** [comment]
Fixed in 9898f1964927bf8782f255cbaabc6acee2c313f3
